README.md

```diff
@@ -1,143 +1,41 @@
-# IntelligentAuto
+# OpenDash
 
-IntelligentAuto is a Qt-based infotainment center for your current Linux OpenAuto installation!
+OpenDash is a Qt-based infotainment center for your Linux OpenAuto installation!
+The OpenDash project includes OpenAuto, AASDK, and Dash.
 
-Main features include:
+Main features of Dash include:
 
-* volume control
-* responsive UI (adjusts to any screen size)
-* embedded OpenAuto
-* bluetooth media control
-* read vehicle OBD-II data
-* theming (dark/light mode and accent color)
-* brightness (simulated)
+*	Embedded OpenAuto `Windowed/Fullscreen`
+*	Wireless OpenAuto Capability
+*	On-screen Volume, Brightness, & Theme Control
+*	Responsive Scalable UI `Adjustable for screen size`
+*	Bluetooth Media Control
+*	Real-Time Vehicle OBD-II Data & SocketCAN Capabilities
+*	Theming `Dark/Light mode` `Customizable RGB Accent Color`
+*	True Raspberry Pi 7” Official Touchscreen Brightness Control
+*	App-Launcher built in
+*	Camera Access `Streaming/Local` `Backup` `Dash`
+*	Keyboard Shortcuts `GPIO Triggerable`
 
-![home](https://github.com/rsjudka/intelligent-auto/blob/master/docs/imgs/home.png)
+![](docs/imgs/opendash-ui.gif)
 
-![openauto_maps](https://github.com/rsjudka/intelligent-auto/blob/master/docs/imgs/openauto_maps.png)
 
-![openauto_spotify](https://github.com/rsjudka/intelligent-auto/blob/master/docs/imgs/openauto_spotify.png)
+# Getting Started
 
-![media](https://github.com/rsjudka/intelligent-auto/blob/master/docs/imgs/media.png)
+## Install Script
 
-![data](https://github.com/rsjudka/intelligent-auto/blob/master/docs/imgs/data.png)
+Dash can be built automatically utilizing an included script.
 
-![settings](https://github.com/rsjudka/intelligent-auto/blob/master/docs/imgs/settings.png)
+The install script included in the dash repo will install all the required packages and compile all portions of the OpenDash project.
 
-## Getting Started
-
-### Install.sh automated install (Raspberry Pi only)
-First you need to run the executable with the following command:
-
-`./install.sh`
-
-Once installed it will start up, to run afterwards, navigate to bin folder and run:
-
-`./dash`
-
-Video walk through: 
```

Comment:
can you leave the link to the video, but maybe just give a disclaimer that the process has changed slightly

Reply:
ia renamed to dash, older ui, etc

Reply:
Updated with Video walkthrough...per comments